### PR TITLE
designed queries

### PR DESCRIPTION
**Pull request by zmzade**

### CODE REVIEW

File: databases/week1/w1.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+SELECT COUNT(task.id) FROM task;
```
Comment (calindurnea) [inline databases/week1/w1.sql]: Please add the comments of each task, it's much easier to follow the intended purpose of each query :)

Comment (calindurnea) [inline databases/week1/w1.sql]: You should create a new line per statement, e.g.
`SELECT COUNT(task.id)`
`FROM task;`

File: databases/week1/w1.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+SELECT * FROM task;
+SELECT COUNT(task.id) FROM task WHERE due_date IS NULL ;
+SELECT * FROM task JOIN status ON task.status_id = status.id WHERE status.name like "done";
+SELECT * FROM task JOIN status ON task.status_id = status.id WHERE status.name not like "done";
```
Comment (calindurnea) [inline databases/week1/w1.sql]: The SQL statements are case sensitive so if the value of the status.name is "Done" then this query will not return the correct value. 

You can also use `= "Done"` and `!= "Done"`

File: databases/week1/w1.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+--5--Get all the tasks, sorted with the most recently created first
+SELECT * FROM task ORDER BY created DESC ;
+--6--Get the single most recently created task
+SELECT * FROM task ORDER BY created DESC LIMIT 1;
+--7--Get the title and due date of all tasks where the title or description contains database
+SELECT title, due_date FROM task where CONCAT(title, description) like "%database%" ;
```
Comment (calindurnea) [inline databases/week1/w1.sql]: The `CONCAT` method creates a single value from the values that are passed, so in this case what would happen is that the title, lets say `Title1` and the description, `Description1`, will be united like this `Title1Description1`.

The solution here should include a `OR` statement, e.g. `WHERE title LIKE '%database%' OR description LIKE '%database%'`

File: databases/week1/w1.sql [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+--8--Get the title and status (as text) of all tasks
+SELECT task.title, status.name FROM task JOIN status on task.status_id = status.id;
+--9--Get the name of each status, along with a count of how many tasks have that status
+SELECT status.name, count(task.status_id) as tasks FROM task JOIN status ON task.status_id=status.id GROUP BY status.name;
+--10--Get the names of all statuses, sorted by the status with most tasks first
+SELECT status.name, COUNT(task.status_id) as tasks FROM task JOIN status ON  task.status_id = status.id GROUP BY status.name ORDER BY tasks DESC
```
Comment (calindurnea) [inline databases/week1/w1.sql]: A more suiting name for `count(task.status_id)` would maybe be `task_count`. e.g `count(task.status_id) as task_count`

**Review by calindurnea — COMMENTED** [review]
Great job  👍 ! I added some comments

File: databases/week1/w1.sql (hunk 3 above)
Comment (zmzade) [reply databases/week1/w1.sql]: So thanks for great comments

File: databases/week1/w1.sql (hunk 1 above)
Comment (zmzade) [reply databases/week1/w1.sql]: I hope I edited this part correctly.

**Review by calindurnea — COMMENTED** [review]
Perfect! Nicely done

**Pull request merged**